CHANGELOG.md

```diff
@@ -0,0 +1,12 @@
+# dbt_xero_source v0.3.0
+
+## Features
+- Allow for multiple sources by unioning source tables across multiple Xero connectors.
+([#11](https://github.com/fivetran/dbt_xero_source/pull/11))
+  - Refer to the README for more details. (See ### Unioning Multiple Xero Connectors).
+
+## Under the Hood
+- The unioning occurs in the tmp models using the `fivetran_utils.union_data` macro.
+- Because columns that were previously used for unique tests may now have duplicate fields across multiple sources, these columns are combined with the new `source_relation` column for unique tests and tested using the `dbt_utils.unique_combination_of_columns` macro. 
```

Comment:
This is a great under the hood update to include in the CHANGELOG. Great call out! 🙌